src/app/services/answer.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+import { Observable, from } from 'rxjs';
+import { filter } from 'rxjs/operators';
+
+import { Answer } from '../models/answer';
+import { ANSWERS } from '../mock-data/mock-answers';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AnswerService {
+  private _randomId: number = Math.floor(Math.random() * 10) + 1;
+
+  constructor() { }
+
+  getAnswer(): Observable<Answer> {
+    const answers = from(ANSWERS);
+    return answers.pipe(
+      filter(answer => answer.id == this._randomId)
```

Comment:
You should pretty much always use `===` instead of `==`. There are very few times that it makes sense to use `==` and usually it's because of some legacy code or something 😉